src/Unleash/Properties/AssemblyInfo.cs

```diff
@@ -1,3 +1,4 @@
 ﻿using System.Runtime.CompilerServices;
 
-[assembly: InternalsVisibleTo("Unleash.Tests")]
\ No newline at end of file
+[assembly: InternalsVisibleTo("Unleash.Tests")]
+[assembly: InternalsVisibleTo("DynamicProxyGenAssembly2")]
```

Comment:
What for?
Without it mocking does not work or what?

Reply:
I was unable to mock `IUnleashApiClient` with fake it easy without letting `DynamicProxyGenAssembly2` see internals.
The fake it easy error message suggested this change